frontend/src/lib/components/CommandPalette/commandPaletteLogic.ts

```diff
@@ -627,28 +634,33 @@ export const commandPaletteLogic = kea<
                 resolver: {
                     icon: CommentOutlined,
                     display: 'Share Feedback',
-                    synonyms: ['send opinion', 'ask question', 'message posthog'],
+                    synonyms: ['send opinion', 'ask question', 'message posthog', 'github issue'],
                     executor: () => ({
                         scope: 'Sharing Feedback',
-                        instruction: "What's on your mind?",
-                        icon: CommentOutlined,
-                        resolver: (argument) => [
+                        resolver: [
                             {
-                                icon: SendOutlined,
                                 display: 'Send Message Directly to PostHog',
-                                executor: !argument?.length
-                                    ? undefined
-                                    : () => {
-                                          window.posthog?.capture('palette feedback', { message: argument })
-                                          return {
-                                              scope: 'Sharing Feedback',
-                                              resolver: {
-                                                  icon: CheckOutlined,
-                                                  display: 'Message Sent!',
-                                                  executor: true,
+                                icon: CommentOutlined,
+                                executor: () => ({
+                                    instruction: "What's on your mind?",
+                                    icon: CommentOutlined,
+                                    resolver: (argument) => ({
```

Comment:
Q: Should we add a type here? Non-obvious as a reader what this does.

Reply:
Not sure I understand where to add a type, could you expand?

Reply:
```typescript
resolver: (argument: ArgumentType) => ({
```

Reply:
Hmm, argument is just a string, which is pretty intuitive IMO.

Reply:
It's not obvious for the reader of the code - in general it would be nice if all methods had signatures. But leaving it up to.

Reply:
👍 Will keep in mind